src/main/java/org/gridsuite/modification/server/dto/UpdateModificationGroupResult.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.modification.server.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * This is the return value of the endpoint updateModificationGroup.
+ * TODO : remove this DTO when modificationFailures will not be needed anymore
```

Comment:
TODO : remove this DTO and return only networkModificationResult when modificationFailures will not be needed anymore

Reply:
Ok